gensim/sklearn_api/lsimodel.py

```diff
@@ -20,14 +15,36 @@
 
 
 class LsiTransformer(TransformerMixin, BaseEstimator):
-    """
-    Base LSI module
+    """Base LSI module.
+
+    Scikit learn interface for `gensim.models.lsimodel` for easy use of gensim with scikit-learn.
+    Follows scikit-learn API conventions.
+
     """
 
     def __init__(self, num_topics=200, id2word=None, chunksize=20000,
                  decay=1.0, onepass=True, power_iters=2, extra_samples=100):
-        """
-        Sklearn wrapper for LSI model. See gensim.model.LsiModel for parameter details.
+        """Sklearn wrapper for LSI model.
+
+        Parameters
+        ----------
+        num_topics : int, optional
```

Comment:
Wdyt about the link to original method only (for avoiding duplication)?

Reply:
I also thought about that and I am not sure what is better. On one hand now we have duplication but on the other hand its easier for the developer and user to see the documentation in one tab. Because not all parameters are propagated to the inner model, some of the parameters will be visible in the wrapper and some in the original model (you would need 2 tabs open). I am a bit in favor of duplicating but not 100% sure so if you prefer I will remove duplication.

Reply:
So, maybe combine both approaches: mentioned parameter & type here, but for description - sent the user to the parameter from original class?

Reply:
Ok that sounds reasonable, I will apply asap

Reply:
@steremma we discuss this questions again and this isn't good idea, because it's OK if user look into documentation online (and have a link), but if user use `python`/`jupyter`, he will call something like `help(model)` or `model?` and for this case, links don't work :( (and this is the main problem). For this reason - can you return descriptions for parameters, copy-paste is the lesser evil than docstring, that exists, but useless, if you can't read it in your interpreter.

Also, the link to original class must be in any case too.